src/main/java/create_task_use_case/CreateTaskInteractor.java

```diff
@@ -0,0 +1,50 @@
+package create_task_use_case;
+
+import entities.Task;
+import entities.TaskFactory;
+
+/**
+ * Task Creation Interactor Class. Implements the createTask method provides
+ * by the input boundary and orchestrates the application logic for the use case.
+ * @author Fardin Faruk
+ */
+
+public class CreateTaskInteractor implements CreateTaskInputBoundary {
+    final CreateTaskDsGateway dsGateway;
+    final CreateTaskOutputBoundary outputBoundary;
+    final TaskFactory taskFactory;
+
+    final private String[] DAYSOFWEEK = {"Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"};
```

Comment:
Should use underscores in between words in the variable name. DAYS_OF_WEEK

Reply:
done